requirements.txt

```diff
@@ -0,0 +1,8 @@
+ghapi
```

Comment:
I'm wondering if it would be worth limiting the versions for the libraries to <major>.<minor>?

Reply:
Minor comments